students/tlugosan/lesson02/fibonacci_and_friends.py

```diff
@@ -0,0 +1,47 @@
+""" Compute and print a generalization of Fibonacci series when the first 2 numbers are custom."""
+
+
+def iter_gen_fibonacci(input_number, first_number, second_number):
+    """Computes Fibonacci numbers when the numbers are parameterized."""
+    fib_list = [first_number, second_number]
+    if input_number <= 1:
+        return fib_list[:input_number + 1]
+    for i in range(2, input_number + 1):
+        temp = fib_list[i - 1] + fib_list[i - 2]
+        fib_list.append(temp)
+    return fib_list
+
+
+def print_iter_gen_fibbonaci(input_number, first_number, second_number):
+    """Prints Fibonacci numbers when the numbers are parameterized."""
+    result_list = iter_gen_fibonacci(input_number, first_number, second_number)
+    print(result_list)
+
+
+def recursive_gen_fibonacci(input_number, first_number, second_number):
+    """Computes Fibonacci numbers when the numbers are parameterized."""
+    if input_number == 0:
+        return first_number
+    if input_number == 1:
+        return second_number
+    return recursive_gen_fibonacci(input_number - 1, first_number, second_number) + recursive_gen_fibonacci(input_number - 2, first_number, second_number)
```

Comment:
I believe your question on multi-line is referring to this line:
```
    return recursive_gen_fibonacci(input_number - 1, first_number, second_number) + recursive_gen_fibonacci(
        input_number - 2, first_number, second_number)
```

You can break lines up after opening bracket. If you use something like Pycharm, it can auto-format the code for you.